src/mozanalysis/experiment.py

```diff
@@ -531,6 +532,43 @@ def _build_enrollments_query_fenix_baseline(self, time_limits):
             dataset=self.app_id or "org_mozilla_firefox",
         )
 
+    def _build_enrollments_query_glean_baseline(self, time_limits, dataset):
```

Comment:
Let's name this `glean_event` since it doesn't use the baseline ping like the fallback method does (no idea why we adopted two different names for it)